### PR TITLE
To Do list list structure:- module 2

### PR DESCRIPTION
**Pull request by yusufyusie**
### In this milestone ES6 modules to write modular JavaScript and webpack to bundle JavaScript.
**The following tasks are included in this project**

- Created an index.html file 
- Created an empty To Do List placeholder
- Created an index.js file and set an array of some simple to do tasks (array of objects).
- Wrote a function to iterate over the tasks array and populate an HTML list item element for each task.
- Created a style.css and set rules for the To Do List.
- All source files (index.html, index.js and style.css) must be located in /src directory and  distribution files will be generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

File: src/index.js
Comment (joskalenda) [inline src/index.js:1]: - [ ] I like your work, but you know it is not asked at this moment. Your pull request states what you did not do, please make you create a static object of tasks that will be rendered dynamically in the HTML file, on page load, your app does not present any data which is normal becuase you did not do it. please reread again the requirement, take a clear look at the image below, you are not asked to adding or removing any task from the todo, all you have is create an array of objects and iterate over it and render the content on page load

`What asked`



`What you did`

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @yusufyusie , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ].. ✅ Readme
- [ ].. ✅ Clean code
- [ ].. ✅ Design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @yusufyusie , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Readme
✅ ... Tasks created

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ridwanediallo** [comment]
Hi @yusufyusie , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🚩 🚩 🚩

You have submitted a project for  a different milestone. Please ensure to submit the right project for the right milestone. :ok_hand: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**